<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QMainWindow" name="MainPage">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Склад</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1600</width>
     <height>900</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="search">
       <attribute name="title">
        <string>Склад</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="1">
         <widget class="QPushButton" name="search_btn">
          <property name="text">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_name_ru">
          <property name="text">
           <string>Наименование(RU)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_invent_num">
          <property name="text">
           <string>Инвентарный номер</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="invent_num"/>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButton_all">
          <property name="text">
           <string>Показать всё</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_name_en">
          <property name="text">
           <string>Наименование(EN)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QTableView" name="tableView"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="part_num"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_manufacturer">
          <property name="text">
           <string>Производитель</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="name_ru"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="name_en"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="manufacturer"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_part_num">
          <property name="text">
           <string>Артикул</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="edit">
       <attribute name="title">
        <string>Управление</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="new_equipment">
          <property name="text">
           <string>Добавить новое оборудование</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_show_all_users">
          <property name="text">
           <string>Показать всех пользователей</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_edit_equipment">
          <property name="text">
           <string>Редактировать имеющееся оборудование</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="new_user">
          <property name="text">
           <string>Добавить нового пользователя</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_delete_equipment">
          <property name="text">
           <string>Удалить оборудование</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_delete_user">
          <property name="text">
           <string>Удалить пользователя</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_edit_story">
          <property name="text">
           <string>Редактировать историю</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_clear_story">
          <property name="text">
           <string>Очистить историю</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Поле для SQL-запроса</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QTextEdit" name="textEdit_SQL">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QTableView" name="tableView_SQL">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_SQL">
          <property name="text">
           <string>Сделать запрос</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="archive">
       <attribute name="title">
        <string>История</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="logs_table"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Помощь по запросам</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Имена таблиц: &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;part_list			- всё оборудование&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;logs			- история использования&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;user_name		- фамилия и имя&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;SELECT * FROM part_list;							- показать всю таблицу с оборудованием.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;SELECT * FROM part_list WHERE lower(name_ru)=lower('ДАтчик');			- показать всё оборудование с наименованием на русском = Датчик, независимо от регистра.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;UPDATE part_list SET act_count=10 WHERE part_num=1233321;			- изменить актуальное количество оборудования с артикулом 1233321.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;DELETE FROM user_name WHERE name_surname='Газизов Азамат';		- удалить из таблицы user_name строку 'Газизов Азамат'.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>name_ru</tabstop>
  <tabstop>name_en</tabstop>
  <tabstop>manufacturer</tabstop>
  <tabstop>part_num</tabstop>
  <tabstop>invent_num</tabstop>
  <tabstop>search_btn</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton_all</tabstop>
  <tabstop>new_equipment</tabstop>
  <tabstop>pushButton_show_all_users</tabstop>
  <tabstop>pushButton_edit_equipment</tabstop>
  <tabstop>new_user</tabstop>
  <tabstop>pushButton_delete_equipment</tabstop>
  <tabstop>pushButton_delete_user</tabstop>
  <tabstop>pushButton_edit_story</tabstop>
  <tabstop>pushButton_clear_story</tabstop>
  <tabstop>textEdit_SQL</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>logs_table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
